scripts/mysql/upgrade-0.5.0-to-0.5.1-mysql.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- Copyright 2024 Datastrato Pvt Ltd.
+-- This software is licensed under the Apache License version 2.
+--
+
+ALTER TABLE `role_meta`
+    ADD COLUMN `securable_objects` VARCHAR(2048) NOT NULL COMMENT 'securable objects' AFTER `properties`;
+
+ALTER TABLE `role_meta` DROP COLUMN `securable_object_full_name`;
+ALTER TABLE `role_meta` DROP COLUMN `securable_object_type`;
+ALTER TABLE `role_meta` DROP COLUMN `privileges`;
+ALTER TABLE `role_meta` DROP COLUMN `privilege_conditions`;
```

Comment:
In upgrade script, we should never remove fields, we should maintain backward compatibility.

If we execute this script, the production environment will immediately report an error.

Reply:
I change to use alter.